<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GQManagerWidget</class>
 <widget class="QMainWindow" name="GQManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Field 3D Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/sg_3dviewer.qrc">
    <normaloff>:/File/application.ico</normaloff>:/File/application.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="SGCanvasWidget" name="mWgtApp" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mMnbMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="mAtnImageOpen"/>
    <addaction name="mAtnSetting"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="mAtnZoomPrev"/>
     <addaction name="mAtnZoomNext"/>
     <addaction name="mAtnRefresh"/>
     <addaction name="separator"/>
     <addaction name="mAtnZoomIn"/>
     <addaction name="mAtnZoomOut"/>
     <addaction name="mAtnZoomToLayer"/>
    </widget>
    <widget class="QMenu" name="menuMeasurement">
     <property name="title">
      <string>Measurement</string>
     </property>
     <addaction name="mAtnViewDistance"/>
     <addaction name="mAtnViewArea"/>
     <addaction name="mAtnViewVolumn"/>
    </widget>
    <widget class="QMenu" name="menu3D_Sight">
     <property name="title">
      <string>3D Sight</string>
     </property>
     <addaction name="mAtnViewSightLinear"/>
     <addaction name="mAtnViewSightPolyhedral"/>
    </widget>
    <addaction name="mAtnViewPan"/>
    <addaction name="menuZoom"/>
    <addaction name="menuMeasurement"/>
    <addaction name="menu3D_Sight"/>
    <addaction name="mAtnTerrainProfile"/>
    <addaction name="mAtnViewPointGrid"/>
    <addaction name="mAtnViewAnimation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mTlbFile">
   <property name="windowTitle">
    <string>File Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mAtnImageOpen"/>
   <addaction name="mAtnDEMOpen"/>
   <addaction name="mAtn3DOpen"/>
  </widget>
  <widget class="QToolBar" name="mTlbView">
   <property name="windowTitle">
    <string>View Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mAtnViewPan"/>
   <addaction name="mAtnViewDistance"/>
   <addaction name="mAtnViewArea"/>
   <addaction name="mAtnViewVolumn"/>
   <addaction name="mAtnTerrainProfile"/>
   <addaction name="separator"/>
   <addaction name="mAtnViewSightRadial"/>
   <addaction name="mAtnViewPointGrid"/>
   <addaction name="mAtnViewAnimation"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="mAtnSetting"/>
  </widget>
  <action name="mAtnImageOpen">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/open_image.png</normaloff>:/File/open_image.png</iconset>
   </property>
   <property name="text">
    <string>Image Open...</string>
   </property>
  </action>
  <action name="mAtnSearch">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/1-48.png</normaloff>:/File/1-48.png</iconset>
   </property>
   <property name="text">
    <string>Search...</string>
   </property>
  </action>
  <action name="mAtnUpload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/File/upload.png</normaloff>:/File/upload.png</iconset>
   </property>
   <property name="text">
    <string>Upload...</string>
   </property>
  </action>
  <action name="mAtnSetting">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/3-48.png</normaloff>:/File/3-48.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="mActExtractBoundary">
   <property name="icon">
    <iconset>
     <normaloff>:/File/boundary.png</normaloff>:/File/boundary.png</iconset>
   </property>
   <property name="text">
    <string>Extract boundary...</string>
   </property>
   <property name="iconText">
    <string>Extract</string>
   </property>
  </action>
  <action name="mAtnDownload">
   <property name="icon">
    <iconset>
     <normaloff>:/File/download.png</normaloff>:/File/download.png</iconset>
   </property>
   <property name="text">
    <string>Download...</string>
   </property>
  </action>
  <action name="mActLocMatch">
   <property name="icon">
    <iconset>
     <normaloff>:/File/locationmatch.png</normaloff>:/File/locationmatch.png</iconset>
   </property>
   <property name="text">
    <string>Location match...</string>
   </property>
   <property name="iconText">
    <string>Match</string>
   </property>
  </action>
  <action name="mAtnSave">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/48_save.png</normaloff>:/File/48_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="mAtnViewPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewPan.png</normaloff>:/View/mAtnViewPan.png</iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
  <action name="mAtnZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/View/mAtnViewZoomInRect_48.png</normaloff>:/View/mAtnViewZoomInRect_48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="mAtnZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/View/mAtnViewZoomOutRect_48.png</normaloff>:/View/mAtnViewZoomOutRect_48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="mAtnZoomToLayer">
   <property name="text">
    <string>Zoom to Selected Layer</string>
   </property>
  </action>
  <action name="mAtnZoomPrev">
   <property name="icon">
    <iconset>
     <normaloff>:/View/mAtnViewZoomPrev_48.png</normaloff>:/View/mAtnViewZoomPrev_48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Previous</string>
   </property>
  </action>
  <action name="mAtnZoomNext">
   <property name="icon">
    <iconset>
     <normaloff>:/View/mAtnViewZoomNext_48.png</normaloff>:/View/mAtnViewZoomNext_48.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Next</string>
   </property>
  </action>
  <action name="mAtnRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="mAtnViewDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewDistance.png</normaloff>:/View/mAtnViewDistance.png</iconset>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
  <action name="mAtnViewArea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewArea.png</normaloff>:/View/mAtnViewArea.png</iconset>
   </property>
   <property name="text">
    <string>Area</string>
   </property>
  </action>
  <action name="mAtnViewVolumn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewVolume.png</normaloff>:/View/mAtnViewVolume.png</iconset>
   </property>
   <property name="text">
    <string>Volumn</string>
   </property>
  </action>
  <action name="mAtnTerrainProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewSectionalPlan.png</normaloff>:/View/mAtnViewSectionalPlan.png</iconset>
   </property>
   <property name="text">
    <string>Terrain Profile</string>
   </property>
  </action>
  <action name="mAtnViewSightLinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewSightLinear.png</normaloff>:/View/mAtnViewSightLinear.png</iconset>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="mAtnViewSightRadial">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewSightCircle.png</normaloff>:/View/mAtnViewSightCircle.png</iconset>
   </property>
   <property name="text">
    <string>Radial</string>
   </property>
  </action>
  <action name="mAtnViewSightPolyhedral">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewSightPoly.png</normaloff>:/View/mAtnViewSightPoly.png</iconset>
   </property>
   <property name="text">
    <string>Polyhedral</string>
   </property>
  </action>
  <action name="mAtnImport">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/48.png</normaloff>:/File/48.png</iconset>
   </property>
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>사용자관리</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>사용자 관리</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>영상 관리</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>영상 이력 관리</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>재해 분석</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>셋팅</string>
   </property>
  </action>
  <action name="mAtnDEMOpen">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/open_dem.png</normaloff>:/File/open_dem.png</iconset>
   </property>
   <property name="text">
    <string>DEM Open...</string>
   </property>
  </action>
  <action name="mAtn3DOpen">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/File/open_3d.png</normaloff>:/File/open_3d.png</iconset>
   </property>
   <property name="text">
    <string>3D Open...</string>
   </property>
  </action>
  <action name="mAtnViewPointGrid">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewPointGrid.png</normaloff>:/View/mAtnViewPointGrid.png</iconset>
   </property>
   <property name="text">
    <string>Z Point Grid</string>
   </property>
   <property name="iconText">
    <string>Point Grid</string>
   </property>
  </action>
  <action name="mAtnViewAnimation">
   <property name="icon">
    <iconset resource="../resource/sg_3dviewer.qrc">
     <normaloff>:/View/mAtnViewAnimation.png</normaloff>:/View/mAtnViewAnimation.png</iconset>
   </property>
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SGCanvasWidget</class>
   <extends>QWidget</extends>
   <header location="global">sg_canvaswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource/sg_3dviewer.qrc"/>
 </resources>
 <connections/>
</ui>
